docs/gpa.md

```diff
@@ -233,78 +231,23 @@ The information included in the notice of intended procurement covers all the in
     <tbody>
       <tr class="section">
         <td>X:7</td>
-        <td colspan="2">A procuring entity shall make available to suppliers tender documentation that includes all information necessary to permit suppliers to prepare and submit responsive tenders. Unless already provided in the notice of intended procurement, such documentation shall include a complete description of:</td>
-      </tr>
-      <tr>
-        <td>X:7(a)</td>
-        <td>the procurement, including the nature and the quantity of the goods or services to be procured or, where the quantity is not known, the estimated quantity and any requirements to be fulfilled, including any technical specifications, conformity assessment certification, plans, drawings or instructional materials;</td>
-        <td markdown=1>
-
-* 
-</td>
-      </tr>
-      <tr>
-        <td>X:7(b)</td>
-        <td>any conditions for participation of suppliers, including a list of information and documents that suppliers are required to submit in connection with the conditions for participation;</td>
-        <td markdown=1>
-
-* 
-</td>
-      </tr>
-      <tr>
-        <td>X:7(c)</td>
-        <td>all evaluation criteria the entity will apply in the awarding of the contract, and, except where price is the sole criterion, the relative importance of such criteria;</td>
-        <td markdown=1>
-
-* 
-</td>
-      </tr>
-      <tr>
-        <td>X:7(d)</td>
-        <td>where the procuring entity will conduct the procurement by electronic means, any authentication and encryption requirements or other requirements related to the submission of information by electronic means;</td>
+        <td>A procuring entity shall make available to suppliers tender documentation that includes all information necessary to permit suppliers to prepare and submit responsive tenders. Unless already provided in the notice of intended procurement, such documentation shall include a complete description of:</td>
         <td markdown=1>
 
-* 
-</td>
+* If at the time of publishing tender documentation (article X:7) you have already published all the information necessary for a notice of intended procurement (article VII:2), and this information has not changed, discard this article.
+* If you haven't published all the information necessary for a notice of intended procurement or if the information has changed, follow the guidance for article VII:2 to publish the new and updated data.
```

Comment:
Are you sure that all parts of X:7 covered by VII:2?

Reply:
Please review the documentType codelist, in case there are documents relevant to the paragraphs of X:7.

Reply:
I've added evaluation critieria docs in 0b2d217 and planned procurement notice in 3021416

Reply:
Done